.vscode/settings.json

```diff
@@ -11,7 +11,13 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "[typescriptreact]": {
-    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "editor.tabSize": 2,
+  "editor.detectIndentation": false,
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
```

Comment:
I might run this by #ui-extensions-api-releases to see what they think since it affects the whole repo. I'm all for it.

Reply:
Sent a message, lets see if they say anything